templates/asc-alert-send-notification-email.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "/providers/Microsoft.Logic/galleries/public/templates/d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "type": "Microsoft.Logic/galleries/templates",
+    "name": "d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "properties": {
+        "author": {
+            "displayName": "Microsoft"
+        },
+        "categoryNames": [
+            "general",
+			"security"
```

Comment:
Indentation (this and the rest of the files)

Reply:
I fixed the indentation